hourly_watch.py

```diff
@@ -54,14 +56,23 @@ def age(self):
                     time_header = 'last-modified'
                 else:
                     time_header = 'date'
-                parsed_time = time.strptime(self.headers[time_header],
-                                            "%a, %d %b %Y %H:%M:%S %Z")
-                # go through time's hoops to work with this in the right timezone
-                os.environ['TZ'] = time.strftime("%Z", parsed_time)
-                time.tzset()
-                self.filetime = time.mktime(parsed_time)
-                del os.environ['TZ']
-                self._age = (time.mktime(time.gmtime()) - self.filetime) / 86400.0
+                # Parse the time including time zone. Make a new datetime object corresponding
```

Comment:
All these changes were driven by the existing code not working correctly and giving a 5.00 hour difference for what should be a 0.00 hour difference.  I decided to just start over to avoid the env `TZ` hack.

Reply:
Was that a Py3 thing or was the old code broken on Py2 and I didn't notice?  I assume for these convoluted hacks (that look like mine) that it worked at some point.

Reply:
I'm pretty sure it was working since the issue I saw resulted in obvious cron job failure warnings. I didn't investigate further.